boards/shields/nrf2240ek/nrf2240ek.overlay

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Pin mapping suitable for use nRF2240-EK PCA63564 with
+ * Nordic Interposer Board A PCA64172 v0.2.0.
+ * The nRF2240-EK in SLOT 2 of PCA64172.
```

Comment:
```suggestion
 * Nordic Interposer Board A PCA63564 v0.2.0.
 * The nRF2240-EK in SLOT 2 of PCA63564.
```

Reply:
Your proposal is incorrect. The Nordic Interposer Board A is PCA64172. The nRF2240-EK goes into SLOT2 of the Interposer Board so PCA64172 here is correct. The PCA63564 is the nRF2240-EK itself not the interposer board.

Reply:
Oops, you're right!